README.md

```diff
@@ -19,194 +19,62 @@
 
-1. Once your project has been initialized, you can push your application to your OpenShift cluster for the first time using the following command:
+1. Push your app to your cluster
 
     ```shell
     odo push
+    odo watch
     ```
 
-    Upon a first time `odo push` invocation, the following occurs:
-    - An Open Shift managed kubernetes pod is launched within the namespace created in step 3 above,
-    - A container based on the image referenced in the devfile is started in that pod
-    - odo syncs your local source code directory to that container. (under the `/projects` directory)
-    - The app is compiled and built within the container and liberty dev mode is started to host the app once built.
-    - A URL is activated and is usable to access the app
-
-1. Retrieve the URL for your app using the odo CLI:
+1. Retrieve the URL for you app
 
     ```shell
     odo url list
     ```
 
-1. You should be able to access the following endpoints, as they are exposed by your template application by default:
-
-    - Readiness endpoint: http://`<odo url>`/health/ready
-    - Liveness endpoint: http://`<odo url>>`/health/live
-    - Metrics endpoint: http://`<odo url>`/metrics (login as `admin` user with password obtained as mentioned [here](#Metrics-Password).
-
-To see source code changes reflected in the running application you can re-issue the `odo push` command after any edit session is complete and saved. Source code changes will be reflected in the running container within a few seconds. Alternatively, you can issue `odo watch` once which will monitor for changes in your local project and automatically push changes to your remote cluster.
-
-## Creating a project from an existing application
-
-You can initialize an existing application as a stack project. The following steps show how to do this using the Open Liberty [application-stack-intro sample application](https://github.com/OpenLiberty/application-stack-intro).
-
-1. Clone the application-stack-intro repository
-
-```shell
-git clone git@github.com:OpenLiberty/application-stack-intro.git
-cd application-stack-intro
-```
-
-2. Initialize the project with the java-openliberty stack
-
-```shell
-odo create java-openliberty my-component
-```
-
-3. Push the sample application to OpenShift
-
-```shell
-odo push
-```
-
-For more details on the application-stack-intro application, check out https://github.com/OpenLiberty/application-stack-intro
-
-## Odo development operations
-
-### PUSH
-By default, `odo push`, in conjuction with the java-openliberty devfile, runs a liberty:dev mode in "hot test" mode, where unit tests and integration tests get automatically re-executed after each detected change. These test results are visible in the OpenShift container logs. To see the realtime contents of the container logs, use the odo CLI:
-```shell
-odo log
-```
+You can now begin developing your app! Changes will be detected and pushed automatically. 
```

Comment:
Maybe it should be:

1. odo push
2. odo log -f  # wait
3. odo url list 
4. hit browser/curl
5. odo watch... Changes will be detected..automatically

Reply:
done - I left out the odo log step since we dont have any tests to run out of the box with the starter